src/modules/RA/Scene/components/SceneFormNameField.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { useForm } from 'react-final-form';
+
+import {
+  TextInput,
+  FormDataConsumer,
+} from 'react-admin';
+
+import { toSlug } from '../../../../utils/react-admin/helper';
+
+const SceneFormNameField = ({ formClassName }) => {
+  const form = useForm();
+
+  return (
+    <FormDataConsumer formClassName={formClassName}>
+      {({ formData, ...rest }) => (
+        <TextInput
+          source="name"
+          label="view.form.name"
+          onChange={value => {
+            if (!value) return;
```

Comment:
The early return seems unnecessary because `value` is the `onChange` event and i think it should not be falsy

Reply:
In some (edge)cases, `onChange` might formerly be called with no argument, making `value.target.value`  to throw an error… 

I don't really know if it could still happens, but In case…